### DISCUSSION TITLE
MPQP webinar

### DISCUSSION THREAD

**Discussion by Henri-ColibrITD** [ColibrITD-SAS/mpqp#37] [Announcements]
Hey  everybody 👋

We are thrilled to announce our first webinar!
During this webinar, we will: 
- do a live demo of MPQP,
- answer your questions on why and how to use it.

It will happen on our [YouTube Channel](https://www.youtube.com/@ColibriTDQuantumInnovations), the 16th of April at 11 AM CET.

See you there !

**Comment by hJaffaliColibritd** [comment]
Yeah, let's meet the 16th, looking forward to see y'all !
